server/lib/contributors.ts

```diff
@@ -131,19 +135,20 @@ const loadContributors = async (collectiveId: number): Promise<ContributorsCache
         ARRAY_AGG(DISTINCT m."role") AS roles,
         ARRAY_AGG(DISTINCT tier."id") AS "tiersIds",
         COALESCE(MAX(m.description), MAX(tier.name)) AS description,
-        (${TotalAmountContributedQuery}) AS "totalAmountDonated"
+        COALESCE(tc."totalAmountDonated", 0) AS "totalAmountDonated"
```

Comment:
If the same profile donated both directly **and** through gift cards, my understanding of this query is that it will return two entries. Shouldn't we remove `totalAmountDonated` from the `group by` and set this to:

```suggestion
        COALESCE(SUM(tc."totalAmountDonated"), 0) AS "totalAmountDonated"
```

Reply:
I tested that and It was returning a bit more than expected in some cases, and I decided to rewrite this focusing on keeping the same value.

Reply:
The issue described here did materialize in the issue, we should go back to this query to make sure we group entries properly: https://github.com/opencollective/opencollective/issues/3804